.github/workflows/semver-checks.yml

```diff
@@ -8,6 +8,8 @@ on:
       - edited
```

Comment:
There's the culprit!

Also, shouldn't we be triggering semver on `push` rather than (only) on `pull_request_target`?

Reply:
nice thanks! and yea I think we want both - fixing :)

Reply:
cc @nicklan in case im missing anything?